source/license.html

```diff
@@ -11,16 +11,30 @@
 	<article>
 	  <header><h2>License</h2></header>
 	  <div class="entry-content">
-	    <p>
-	    This is a copy of the current LICENSE file from
-	    the main repository.
-	    The plain-text document version of this document is available
-	    here:
-	    <a href="license.txt">license.txt</a>
-	    </p>
-	    <pre>
-	    <!--#include virtual="license.txt" -->
-	    </pre>
+            <p>
+            OpenSSL is covered by one of two licenses, depending on which
+            release is involved.  In all cases, there is a file named
+            <code>LICENSE</code> in the top-level of the release.
+            Copies can also be found here.
+            </p>
+            <p>
+            For the 3.0.0 release, and later releases derived from that,
+            <a href="apache-license-2.0.txt">the Apache License v2</a>
+            applies.
+            This also applies to the git "master" branch.
+            </p>
+            <p>
+            For any release made before OpenSSL 3.0.0 (namely
+            the 1.1.1, 1.1.0, 1.0.2, and all prior releases including
+            those not currently supported),
+            <a href="license-ssleay.txt">the OpenSSL license</a>
```

Comment:
Technically both the OpenSSL license *and* the SSLeay license applies.

Reply:
I can fix that